### PR TITLE
Added "proxy support" via environment variables

### PR DESCRIPTION
**Pull request by mspike**
Useful if you use proxy in your environment.

Please pull, thanks!
Br,
 Peter

### CODE REVIEW

File: Log/lambda_function.py [hunk 1]
```diff
@@ -36,9 +36,16 @@
     "ddsourcecategory": "aws",
 }
 
+try:
+    host = os.environ['DD_URL']
```
Comment (seittema) [inline Log/lambda_function.py]: Just an observation, you can simplify the code by using os.getenv and setting the default if it's missing
`host = os.getenv("DD_URL", default="lambda-intake.logs.datadoghq.com")`

**Comment by mspike** [comment]
Awwww, pretty awsome. Please do so and make a pull request!

**Comment by NBParis** [comment]
@mspike Do you want to update the current PR or to merge this one and open a new one?

**Comment by mspike** [comment]
@NBParis did it in the PR. Please check and approve.

@seittema thanks for the tip, pretty neat that we don't need an extra try & catch !

**Pull request merged**